packages/server/src/utils/KeyvDocumentNodeLRU.ts

```diff
@@ -0,0 +1,49 @@
+// Create ~about~ a 30MiB LRU. This is less than precise
+// since the technique to calculate the size of a DocumentNode is
+// only using JSON.stringify on the DocumentNode (and thus doesn't account
+// for unicode characters, etc.), but it should do a reasonable job at
+// providing a caching document store for most operations.
+//
+
+import LRUCache from 'lru-cache';
+import Keyv, { Store, type Options } from 'keyv';
+import type { DocumentNode } from 'graphql';
+
+// LRUCache wrapper to implement the Keyv `Store` interface.
+export class LRU<V> extends LRUCache<string, V> implements Store<V> {
+  async delete(key: string): Promise<boolean> {
+    try {
+      super.del(key);
```

Comment:
what kind of errors are we catching here?

Reply:
Not needed, not really sure why I put a try here to begin with.
1ea2820ed1efb34501decb76d36496c8385825f9